vpr/src/route/clock_fwd.h

```diff
@@ -4,6 +4,7 @@
 struct Coordinates {
     int x = -1;
     int y = -1;
+    int layer = 0;
```

Comment:
Change from 0 to -1.
Any other data structure that already has these 3 members?  If so could just switch to that and remove this struct